avalon/pipeline.py

```diff
@@ -216,24 +216,40 @@ def remove(self, container):
 @lib.log
 class Creator(object):
     """Determine how assets are created"""
-    name = None
     label = None
     family = None
+    variants = None
```

Comment:
I quite like that definition where `subste` = `family` + `variant` we use it that way and talk about it that way in the studio, but i never though of specifying it like the. @mottosso what do you think?

Reply:
Note that Variants was coined before, see: https://github.com/getavalon/core/issues/269

Reply:
I think I would pick `defaults` for now, since it's indeed the default names for subsets, and would also make sense if one describe it as *default variants*.

Reply:
> Variants was coined before

right. that's because we use it internally among artists :)

> I think I would pick defaults for now

that works nicely I don't mind either way.